desc/objectives/_fast_ion.py

```diff
@@ -305,50 +314,57 @@ def compute(self, params, constants=None):
         return constants["transforms"]["grid"].compress(data[self._key])
 
     def _build_spline(self, use_jit=True, verbose=1):
+        Y_B = self._hyperparam.pop("Y_B")
+        num_transit = self._hyperparam.pop("num_transit")
+        num_quad = self._hyperparam.pop("num_quad")
+        del self._constants["X"]
```

Comment:
why the deletion?

Reply:
it's just not used so I wanted to delete it.